packages/fuels-core/src/code_gen/custom_types_gen.rs

```diff
@@ -261,19 +261,44 @@ pub fn expand_custom_enum(enum_name: &str, prop: &Property) -> Result<TokenStrea
             // Elementary type
```

Comment:
Document that this includes the "weird" `String(x)` Sway type and the `Arrays` because it is not what someone would immediately think about when reading "elementary type"